include/functions_user.inc.php

```diff
@@ -1168,7 +1168,7 @@ function pwg_login($success, $username, $password, $remember_me)
 ;';
 
   $row = pwg_db_fetch_assoc(pwg_query($query));
-  if (isset($row['id']) and $conf['password_verify']($password, $row['password'], $row['id']))
+  if (isset($row['id']) and $conf['password_verify'](stripslashes($password), $row['password'], $row['id']))
```

Comment:
Very bad idea ! You should use mysql_real_escape_string at least

Reply:
No, because its doesn't generate a SQL request. The given password is passed to the hash function and compared to the hashed password that is stored in the database. The hash function has to get the password without slashes.

But maybe you prefer to put the stripslashes in password_verify function.